charts/redis-cluster/README.md

```diff
@@ -104,6 +104,7 @@ helm delete <my-release> --namespace <namespace>
 | redisExporter.image | string | `"quay.io/opstree/redis-exporter"` |  |
 | redisExporter.imagePullPolicy | string | `"IfNotPresent"` |  |
 | redisExporter.resources | object | `{}` |  |
+| redisExporter.securityContext | object | `{}` | Security context for redis exporter container |
```

Comment:
We generate the README.md file using helm-docs. Please run helm-docs locally. :)

Reply:
Updated!